package-lock.json

```diff
@@ -1,8 +1,9747 @@
 {
   "name": "@livechat/ds-workspace",
   "version": "1.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
You have used a newer version of npm while installing dependencies which caused the switch from lock version 1 to 2. As there are no changes in dependencies in this PR it would be easiest to just omit the changes to this file completely.

Reply:
In fact, I shouldn't have any changes in package-lock. Nothing new added, I will checkout the `v1` version